pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -205,7 +218,7 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				},
 
 				// backwards compatibility
-				authorizationapi.NewRule(read...).Groups(buildGroup).Resources("buildlogs").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(buildGroup, legacyBuildGroup).Resources("buildlogs").RuleOrDie(),
```

Comment:
This is denoted as `backwards compatibility` so I am not sure if it should be updated.  cc @deads2k